TF_Settings_Web/src/Pages/Interactions/InteractionsScreen.tsx

```diff
@@ -1,18 +1,26 @@
-import './Interactions.scss';
+import classnames from 'classnames/bind';
 
-import { Component } from 'react';
+import styles from './Interactions.module.scss';
+
+import { useEffect, useReducer, useState, CSSProperties } from 'react';
 
 import { ConfigurationManager } from 'TouchFree/src/Configuration/ConfigurationManager';
-import { InteractionConfigFull, TrackedPosition } from 'TouchFree/src/Configuration/ConfigurationTypes';
-import { ConfigState, WebSocketResponse } from 'TouchFree/src/Connection/TouchFreeServiceTypes';
+import {
+    InteractionConfig,
+    InteractionConfigFull,
+    TrackedPosition,
+} from 'TouchFree/src/Configuration/ConfigurationTypes';
 import { InteractionType } from 'TouchFree/src/TouchFreeToolingTypes';
 
 import { InteractionExplainer } from '@/Videos';
 
 import { DocsLink, LabelledToggleSwitch, RadioGroup, RadioLine, Slider, TextSlider } from '@/Components';
+import { MiscTextButton } from '@/Components/TFButton/TFButtons';
 
 import { DefaultInteractionConfig } from './SettingsTypes';
 
+const classes = classnames.bind(styles);
```

Comment:
I've only been using it where I need the "conditional classes" feature. Could put it in everywhere if you'd prefer that?

Reply:
My preference would be to bind everywhere for consistency and incase you need add conditional classes later. This would add a few lines of bloat to every file though. Could we make a nicer hook of some sort....?